sourcegraph/about
# Add resources index page

Adds a `resources` index page and closes #5234 

## To Test 
- Visit `/resources` and check that links work and card component looks correct.
